src/index.html

```diff
@@ -1,13 +1,43 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img class="card__image"
+      src="images/imac.jpeg" alt="pc image">
+      <div class="card__title">
+        <span class="title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</span>
+      </div>
+      <span class="code">Product code: 195434</span>
+      <div class="card__rating">
+        <div class="stars stars--4">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
+        </div>
+        <span class="card__reviews">Reviews: 5</span>
+      </div>
+      <div class="price card__pricing">
```

Comment:
In the BEM convention, it's common to structure classes in a more hierarchical manner, like block__element--modifier. In your example, you have combined two block names (price and card) into a single class, which is not the standard BEM format.